sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -518,7 +544,7 @@ export class KeyClient {
 
       return this.getKeyFromKeyBundle(response);
     } else {
-      const response = await this.client.importKey(this.vaultUrl, name, key, options);
+      const response = await this.client.importKey(this.vaultUrl, name, { key });
```

Comment:
I removed "options" since by this point options is undefined. See the if conditional above in the code (you should be able to see the `} else  {` part from here).